### PR TITLE
More tweaks to Ch 4

### PR DESCRIPTION
**Pull request by stefanv**
This should more-or-less address https://github.com/elegant-scipy/elegant-scipy/issues/132

---
This change is [Reviewable](https://reviewable.io/reviews/elegant-scipy/elegant-scipy/213)

### CODE REVIEW

File: markdown/ch4.markdown [hunk 1]
```diff
@@ -291,7 +290,7 @@ freqs, times, Sx = signal.spectrogram(audio, fs=rate, window='hanning',
                                       detrend=False, scaling='spectrum')
 
 ## Plot using:
-# plt.pcolormesh(times, freqs, 10 * np.log10(Sx), cmap='viridis');
+# plt.pcolormesh(times, freqs, 10 * np.log10(Sx));
```
Comment (stefanv) [inline markdown/ch4.markdown]: Because I don't want to plot the colormesh again (already did just above using our calculation).

**Comment by stefanv** [comment]
Look, Wercker is back :)

File: markdown/ch4.markdown (hunk 1 above)
Comment (jni) [inline markdown/ch4.markdown]: I'd say it's worth plotting. It illustrates that it is the same thing. (This was not clear to me, even reading the context. I think it doesn't help that you used imshow above and pcolormesh here.)

Comment (stefanv) [inline markdown/ch4.markdown]: That's why I added the comment, because you can't just do an imshow with scipy.  But, OK, let me uncomment that.

**Comment by jni** [comment]
Hmm, the wercker error is weird — I think you have to pip install -U cycler manually. Maybe some crappy way in which pip or mpl handle dependencies.

**Comment by stefanv** [comment]
Or just merge my other PR in which I bump the mpl requirement :)

**Comment by stefanv** [comment]
Hopefully this will now build after the rebase.

**Comment by jni** [comment]
Nope, now there's a totally different error, LOL

**Comment by hdashnow** [comment]
Just very minor... in the technical review I changed all the discrete fourier transforms to lowercase (except where first defining the acronym DFT).

**Comment by hdashnow** [comment]
Also, now the DFT and FFT acronyms have been defined twice.

**Comment by stefanv** [comment]
If, by define, you mean showing that DFT == Discrete Fourier Transform, then yes.  But I think that's probably not too bad; the reader might miss it the first time around.

**Comment by stefanv** [comment]
Casing fixed.

**Comment by stefanv** [comment]
I still have a few more fixed to add here.

**Comment by jni** [comment]
@stefanv is this ready for review now?

**Comment by stefanv** [comment]
I still have to incorporate LV's comments (partially done).

**Comment by stefanv** [comment]
I'm done for now, thanks.

File: markdown/ch4.markdown [hunk 2]
```diff
@@ -435,8 +480,10 @@ another real-world problem: analyzing target detection in radar data.
 
 ### Frequencies and their ordering
 
+
 For historical reasons, most implementations return an array where
-frequencies vary from low-to-high-to-low.  E.g., when we do the real
+frequencies vary from low-to-high-to-low (*SEE ALSO: Sidebox on DFT
+for further explanation of frequencies).  E.g., when we do the real
```
Comment (jni) [inline markdown/ch4.markdown]: Missing close emphasis tag? =)

File: markdown/ch4.markdown [hunk 3]
```diff
@@ -1075,22 +1148,26 @@ positive or a negative frequency. This component of the DFT represents
...
 The FFT in turn is simply a special and highly efficient algorithm for
 calculating the DFT. Whereas a straightforward calculation of the DFT
 takes of the order of $N^{2}$ calculations to compute, the FFT
 algorithm requires of the order $N\log N$ calculations. The FFT was
 the key to the wide-spread use of the DFT in real-time applications
-and was included in a list of the top 10 algorithms of the $20^{th}$
+and was included in a list of the top $10$ algorithms of the $20^{th}$
 century by the IEEE journal Computing in Science & Engineering in the
-year 2000.
+year $2000$.
```
Comment (jni) [inline markdown/ch4.markdown]: I don't think math font is required for the year or for inline numbers not referring to the mathematics.

**Comment by stefanv** [comment]
My dad complained that the numbers were rendered with the wrong font in
HTML, so... ;)  I'm sure the final LaTeX will be just fine.

**Pull request merged**